packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -771,6 +771,54 @@ void main() {
     await checkCursorBlinking();
   });
 
+  testWidgets('Turning readOnly on stops the cursor', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/108187 .
```

Comment:
Nit: Extra space before the period here.  Or maybe that's fine to distinguish the URL...

Reply:
Yeah when I click the URL in vscode I think it would bring me to a nonexistent page if the space is removed. It's valid in URLs I think.

Reply:
Well I forgot to update the name of the test. Removed the space and updated the test name.